cloud/gcp/storage/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "region" {
+  type        = string
+  description = "Default region for bucket resources"
+}
+
+variable "company-name" {
+    type = string
+    description = "Company name to be used as prefix or postfix"
```

Comment:
question (blocking): is it possible to be consistent and only do either prefixing or postfixing with this variable?

Reply:
question (blocking): would there be benefits to it?

Reply:
Just consistency for human and machine readers. Searching some corpus (Azure or config code or logs or whatever) for a given prefix or postfix would be easier if it were guaranteed to be one or the other